packages/twenty-server/src/engine/core-modules/user/dtos/workspace-member.dto.ts

```diff
@@ -29,4 +33,13 @@ export class WorkspaceMember {
 
   @Field({ nullable: false })
   locale: string;
```

Comment:
as discussed, it would have been nice to update the other fields to Select fields for consistency (locale, colorScheme, timezone)

Reply:
Didn't face any problems with locale or colorScheme, the timezone however seems a little tricky -
1. we have timezones like `Etc/GMT+12`, graphql wont allow us to create these as enums since these include special characters (we can decide to skip these) 
2. Some of the formats are like`US/Indiana-Starke`, `Pacific/Port_Moresby` , `America/North_Dakota/New_Salem` converting these into enum will result in putting `_` on every instance of `/`, `_` or `-`. that would loose its original format, do we want to skip these as well?

Reply:
We could generate the option with name/label values with ChatGPT but what I'm worried about is the size of the enum, not sure if it's worth sending such a large payload in metadata. 

<img width="1213" alt="Screenshot 2024-07-16 at 11 06 43" src="https://github.com/user-attachments/assets/5921f37e-f81a-47b1-b3c8-ba95941aad56">

But you're right it's probably not worth doing it, let's forget it